utils/src/main/java/org/corfudb/utils/lock/LockClient.java

```diff
@@ -82,31 +82,31 @@
     //TODO need to determine if the application should provide a clientId or should it be internally generated.
     public LockClient(UUID clientId, CorfuRuntime corfuRuntime) throws NoSuchMethodException, IllegalAccessException, InvocationTargetException {
 
-        this.taskScheduler = Executors.newScheduledThreadPool(1, (r) ->
+        this.taskScheduler = Executors.newScheduledThreadPool(1, r ->
```

Comment:
please rename `r` to something meaningful

Reply:
done